questionnaire/questionnaire/questionnaire.test.js

```diff
@@ -344,6 +526,8 @@ const questionnaireDefinition = {
         'p-applicant-british-citizen-or-eu-national',
         'p-applicant-theme-undefined',
         'p-applicant-theme-not-found',
+        // 'p--has-legal-authority',
+        // 'p-applicant-are-you-18-or-over',
```

Comment:
I this to explicitly explain that a test should fail due the absence of these array items?

Reply:
No! Some erroneous comments. I'll remove those 👍